sycl/test/fpga_tests/fpga_ac_int.cpp

```diff
@@ -0,0 +1,938 @@
+//==---- fpga_ac_int.cpp - SYCL FPGA Algorithmic C Data Type header test ---==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
```

Comment:
Compile and run lines are missing

Reply:
Thanks for adding these! I was unsure about the exact commands needed.